aws/src/main/java/org/apache/iceberg/aws/glue/AssumeRoleRequestIdentity.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aws.glue;
+
+import java.util.Objects;
+import software.amazon.awssdk.services.sts.model.Tag;
+
+public class AssumeRoleRequestIdentity {
+  private String roleArn;
+  private String sessionName;
+  private String externalId;
+  private Tag tag;
+  private String region;
+  private int timeout;
+
+  public AssumeRoleRequestIdentity(String roleArn, String sessionName,
+      String externalId, Tag tag, String region, int timeout) {
+    this.roleArn = roleArn;
+    this.sessionName = sessionName;
+    this.externalId = externalId;
+    this.tag = tag;
+    this.region = region;
+    this.timeout = timeout;
+  }
+
+  public String getRoleArn() {
```

Comment:
nit: prefer to use method name without `get` in Iceberg, e.g. `roleArn` instead of `getRoleArn`